desktop/cmp/input/JsonInput.ts

```diff
@@ -22,8 +22,8 @@ export const [JsonInput, jsonInput] = hoistCmp.withFactory<JsonInputProps>({
     className: 'xh-json-input',
     render(props, ref) {
         return codeInput({
-            linter: linter,
-            formatter: fmtJson,
+            linter,
+            formatter: safeFmtJson,
```

Comment:
Unsure if we want this? We create a "safe" version of `fmtJson` (https://github.com/xh/hoist-react/pull/3937/files#diff-1bd2a34b6889e1254a813e22f3fecee2169724007549f9c6d2396658c54ebc7bR76). I made this update to use that for jsonInput, but that may create issues I am not thinking of?